src/lib/services/charge-versions.js

```diff
@@ -117,10 +116,12 @@ const getStartDate = chargeVersion => chargeVersion.dateRange.startDate
  * @param {Array<ChargeVersion>} chargeVersions
  */
 const refreshEndDates = chargeVersions => {
-  const filtered = chargeVersions.filter(isCurrentChargeVersion)
-  const sorted = sortBy(filtered, getStartDate)
+  const filteredAndSorted = chargeVersions.filter(isCurrentChargeVersion)
+  filteredAndSorted.sort((a, b) => {
+    return new Date(getStartDate(a)) - new Date(getStartDate(b))
+  })
```

Comment:
It might be a little nicer to chain `.sort()` instead of assigning the filtered value to then mutating it, and we probably want to be explicit about what we're passing in the `.filter()`):
```suggestion
  const filteredAndSorted = chargeVersions
    .filter((chargeVersion) => {
      return isCurrentChargeVersion(chargeVersion)
    })
    .sort((a, b) => {
      return new Date(getStartDate(a)) - new Date(getStartDate(b))
    })
```